### DISCUSSION TITLE
How to have chrome.google.com with dark mode ?

### DISCUSSION THREAD

**Discussion by oeloo** [darkreader/darkreader#8343] [General]
Hi,
Dark reader is great to have all websites in dark, as a consequence when coming across a white websites left, a real pain. Unfortunately so is it with https://chrome.google.com/ and local URL, which I have to use frequently.
I have seen this limitation is due to Chrome not allowing extensions on these URL :(
I have found chrome://flags#enable-force-dark is turning those URL to dark mode. Unfortunately it is also forcing all websites to dark, which causes 2 important issues:
- It is not possible to go temporarily to light mode as it is sometimes needed.
- It is impossible to print any more, your paper is filled with black, running out your printer ink immediaetly

How I darken chrome.google.com & local URL while using Dark Reader for all websites?
Is is possible to limit chrome://flags#enable-force-dark effect to these URLs?

**Comment by Gusted** [comment]
> **How I darken chrome.google.com and use Dark Reader for all websites?

Chrome doesn't allow that. It's a "restricted site" and chrome has no way of disabling that.

**Comment by oeloo** [comment]
Is is possible to limit chrome://flags#enable-force-dark effect to Chrome webstore URL and local URL?

**Reply by Gusted** [reply]
I don't think so, I'm not a chromium developer.

**Comment by oeloo** [comment]
That would be really useful, because when you go the dark mode route, ALL must be dark for your eyes not just part of it.
The more you use dark mode, the more full white is a pain.

**Comment by goyalyashpal** [comment]
hi! i tried searching for chrome web store in the discussions. but didn't find any other related one.
the _STUPID BLINDINGLY_ white color of chrome web store is painful to work with. 

site link: https://chromewebstore.google.com/

and this setting:
* "--- on restricted pages" < Advanced < Dark Reader Settings (`/ui/options/index.html`)

says following things:
```
     'You should enable it in browser flags too' (on enabled)
     'Disabled for web store and other pages' (on disabled)
```

so, i was wondering which flags to enable for that?


https://github.com/darkreader/darkreader/blob/2402a2449e33970825a8c41cb9592ca2206fae9f/src/ui/options/advanced/enable-for-protected-pages.tsx#L14-L17
